### PR TITLE
Updating the data saved in database

### PR DESCRIPTION
**Pull request by cbroadley-ch**
Updating company details to only store company name and company number

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/controllers/features/limited/isThisYourCompanyController.ts [hunk 1]
```diff
@@ -7,11 +7,12 @@ import { COMPANY_DETAILS, SUBMISSION_ID, USER_DATA, GET_ACSP_REGISTRATION_DETAIL
 import { saveDataInSession } from "../../../common/__utils/sessionHelper";
 import { getAcspRegistration } from "../../../services/acspRegistrationService";
 import logger from "../../../utils/logger";
-import { AcspData, Company } from "@companieshouse/api-sdk-node/dist/services/acsp";
+import { AcspData } from "@companieshouse/api-sdk-node/dist/services/acsp";
 import { ErrorService } from "../../../services/errorService";
 import { CompanyDetailsService } from "../../../services/company-details/companyDetailsService";
 import { isThisYourCompanyAnswers } from "../../../services/checkYourAnswersService";
 import { AcspDataService } from "../../../services/acspDataService";
+import { Company } from "../../../model/Company";
```
Comment (sampankumar) [inline src/controllers/features/limited/isThisYourCompanyController.ts:15]: We would still want to use the Company from the api-sdk-node

Comment (sampankumar) [reply src/controllers/features/limited/isThisYourCompanyController.ts:15]: Not sure why this was marked resolved. I think the Company type should be used from data model coming from api-sdk-node

File: src/services/checkYourAnswersService.ts [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import { Session } from "@companieshouse/node-session-handler";
 import { ANSWER_DATA } from "../common/__utils/constants";
 import { Answers } from "../model/Answers";
-import { AcspData, Company } from "@companieshouse/api-sdk-node/dist/services/acsp";
+import { AcspData } from "@companieshouse/api-sdk-node/dist/services/acsp";
 import { saveDataInSession } from "../common/__utils/sessionHelper";
 import { Request } from "express";
+import { Company } from "../model/Company";
```
Comment (sampankumar) [inline src/services/checkYourAnswersService.ts:7]: Same as above

File: src/services/company-details/companyDetailsService.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { Request } from "express";
 import { COMPANY_DETAILS, COMPANY_NUMBER } from "../../common/__utils/constants";
 import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
 import { Session } from "@companieshouse/node-session-handler";
-import { Company } from "@companieshouse/api-sdk-node/dist/services/acsp/types";
+import { Company } from "../../model/Company";
```
Comment (sampankumar) [inline src/services/company-details/companyDetailsService.ts:5]: Same as above

File: src/controllers/features/limited/isThisYourCompanyController.ts (hunk 1 above)
Comment (ttingle-ch) [reply src/controllers/features/limited/isThisYourCompanyController.ts:15]: Company type from api-sdk-node does only contains companyName and companyNumber. Here it is the company type used to show the other values on is this your company screen which contains status etc

**Review by sampankumar — APPROVED** [review]

**Review by abhiahirv1 — APPROVED** [review]

**Pull request merged**